yarn.lock

```diff
@@ -23344,12 +23344,15 @@ __metadata:
   languageName: node
   linkType: hard
 
-"zundo@npm:^2.0.0-beta.19":
-  version: 2.0.0-beta.19
-  resolution: "zundo@npm:2.0.0-beta.19"
+"zundo@npm:^2.0.0-beta.20":
+  version: 2.0.0-experimental.0
```

Comment:
Here's where the resolution is not correct, probably because of the version name.

Reply:
Just in case, I raised charkour/zundo#114 just in case is happening to someone else :smiley: